R/primerTree.R

```diff
@@ -264,3 +267,118 @@ labeled_quantile = function(x, labels, ...){
   names(res) = labels
   res
 }
+
+#' @title calc_rank_dist_ave
+#' Summarize 
+#' pairwise differences.
+
+#' @param x a primerTree object 
+#' @param ... Ignored options
```

Comment:
If `...` are not used you should remove them as typos in argument names will be silently ignored otherwise.